src/assets/projects-logos/tracee.png

```diff
@@ -0,0 +1,980 @@
+
```

Comment:
This file appears to be an HTML file instead of a PNG. Maybe a wget/curl of the `blob` GitHub URL instead of the `raw` URL?

Reply:
oops :)